index.html

```diff
@@ -4,134 +4,162 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        
-        <!-- adding required angular JS library and angular JS script-->
-        <script src="js/angular.min.js"></script>
-        <script src="js/gameapp.js"></script>
-        
         <!--Code for jQuery and bootstrap-->
         <script src="js/jquery-3.1.1.min.js"></script>
         <script src="js/bootstrap.min.js"></script>
+        <!-- adding required angular JS library and angular JS script-->
+        <script src="js/angular.min.js"></script>
+        <script src="js/gameapp.js"></script>
+        <script src="js/recentReleases.js"></script>
+        <script src="js/releaseComponent.js"></script>
```

Comment:
I recommend you use a javascript uglifier and then put all of your JS code in one file, so there does not have to be some many requests to the server. To do this you can use a product like webpack or browserify. But maybe that should be part of the next review.